app/bundles/PageBundle/Model/PageModel.php

```diff
@@ -1038,6 +996,77 @@ public function getPageList($limit = 10, \DateTime $dateFrom = null, \DateTime $
         return $q->execute()->fetchAll();
     }
 
+    /**
+     * Get all params (e.g. UTM tags) from a url.
+     */
+    protected function getQueryFromUrl(string $pageUrl): array
```

Comment:
I see no reason for protected. Private has the great benefit that we can make "BC" changes and won't break anything.
```suggestion
    private function getQueryFromUrl(string $pageUrl): array
```

Reply:
In this way we can not change it in a subclass (inherited). But I see your point and I will change it.